Makefile

```diff
@@ -133,12 +135,13 @@ scratch: docs
 	@rm -rf pykickstart-$(VERSION).tar.gz
 	@rm -rf /tmp/pykickstart-$(VERSION) /tmp/pykickstart
 	@dir=$$PWD; cp -a $$dir /tmp/pykickstart-$(VERSION)
+	@cp /tmp/pykickstart-$(VERSION)/docs/_build/text/kickstart-docs.txt /tmp/pykickstart-$(VERSION)/docs
 	@cd /tmp/pykickstart-$(VERSION) ; $(PYTHON) setup.py -q sdist
 	@cp /tmp/pykickstart-$(VERSION)/dist/pykickstart-$(VERSION).tar.gz .
 	@rm -rf /tmp/pykickstart-$(VERSION)
 	@echo "The archive is in pykickstart-$(VERSION).tar.gz"
 
-rc-release: scratch-bumpver scratch pykickstart.spec
+rc-release: scratch-bumpver scratch pykickstart.spec sign
```

Comment:
I don't like requiring signing for rc builds. It makes it harder for someone who doesn't have gpg setup, and also increases the changes of accidentally releasing signed code by accident.

In weldr-client I made signing conditional (on by default), we should do something similar here - https://github.com/osbuild/weldr-client/blob/main/weldr-client.spec.in

Reply:
Fixed